docs/development.md

```diff
@@ -22,6 +22,106 @@ python -m pip install --upgrade pip
 python -m pip install -r requirements.txt
 ```
 
+## Docker Builds
+
+We have preliminary support for building with Docker images. This is a new
+flow and we would like your feedback on how it works for you and please
+feel free to file any feedback or issues.
+
+Install [Docker Engine](https://docs.docker.com/engine/install/ubuntu/). You don't need Docker Desktop.
+
+You have three types of builds selectable with the Environment Variable `TM_PACKAGES`:`torch-mlir` the
+Release build, `out-of-tree` where torch-mlir is build with a pre-built MLIR and `in-tree` where torch-mlir
+is built as part of the LLVM project along with MLIR.
+
+We mount a ccache and pip cache inside the docker container to speed up iterative builds. Iterative
```

Comment:
Curious, where and how is this pip cache configured? I only saw the ccache directory mounted within the container.

Reply:
Unlike the release build that still runs as root:root I run the CI as the user. So when we mount the home directory you get the pip cache too.